packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -335,6 +345,10 @@ export class GarbageCollector implements IGarbageCollector {
 		snapshotData: IGarbageCollectionSnapshotData | undefined,
 		currentReferenceTimestampMs: number,
 	) {
```

Comment:
So is it now true that this function is only concerned with the GC State (references), not the TS/Deleted lists?